packages/headless/doc-parser/use-cases/search.ts

```diff
@@ -414,6 +414,9 @@ const actionLoaders: ActionLoaderConfiguration[] = [
   {
     initializer: 'loadSortCriteriaActions',
   },
+  {
+    initializer: 'loadProductListingSortActions',
```

Comment:
The `loadProductListingSortActions` should in a separate config file dedicated to the PL use-case,